features/polyExplorer/src/screens/entityDetails/entityDetails.jsx

```diff
@@ -66,9 +66,14 @@ const EntityDetails = () => {
                             </p>
                         ) : null}
                         <div className="location-map">
-                            <h2>
-                                {i18n.t("entityDetailsScreen:jurisdiction")}
-                            </h2>
+                            <div className="separator-unit">
+                                <div className="partial-separator" />
```

Comment:
Do you need a div for this?

Reply:
I think it is actually needed as a container for the title and the lines. I would say the issue here is more about the _naming_ of the classes that could be misleading. It's not exactly a separator, but a title. So what about something like `entity-details-title` to the whole `separator-unit` and just `line` or `title-line` to the `partial-separator`? (Not sure if "line" is too generic, but if it's after `entity-details-title` shouldn't be a problem)

Reply:
(This might be related to the utility class that Richard mentioned in another comment)

Reply:
Well... yes actually I guess I should take over Richard's SectionTitle-component here. Will have a look at it today